src/test/java/teammates/test/cases/datatransfer/AccountAttributesTest.java

```diff
@@ -139,6 +141,78 @@ public void testLegacyAccountEntityToAttributes() {
 
     }
 
+    @Test
+    public void testBuilderWithDefaultValues() {
+        AccountAttributes observedAccountAttributes = AccountAttributes.builder().build();
+
+        assertNull(observedAccountAttributes.createdAt);
+        assertNull(observedAccountAttributes.getEmail());
+        assertNull(observedAccountAttributes.getGoogleId());
+        assertNull(observedAccountAttributes.getInstitute());
+        assertFalse(observedAccountAttributes.isInstructor());
+        assertNull(observedAccountAttributes.getName());
+        assertNull(observedAccountAttributes.studentProfile);
+    }
+
+    @Test
+    public void testBuilderWithPopulatedFieldValues() {
+        String expectedGoogleId = "dummyGoogleId";
+        String expectedEmail = "dummyEmail";
+        String expectedName = "dummyName";
+        String expectedInstitute = "dummyInstitute";
+        boolean expectedIsInstructor = true; //since false case is covered in default test
+        Date expectedCreatedAt = new Date(98765);
+
+        AccountAttributes observedAccountAttributes = AccountAttributes.builder()
+                .withGoogleId(expectedGoogleId)
+                .withEmail(expectedEmail)
+                .withName(expectedName)
+                .withInstitute(expectedInstitute)
+                .withIsInstructor(expectedIsInstructor)
+                .withCreatedAt(expectedCreatedAt)
+                .withDefaultStudentProfileAttributes(expectedGoogleId)
+                .build();
+
+        assertEquals(expectedGoogleId, observedAccountAttributes.getGoogleId());
+        assertEquals(expectedEmail, observedAccountAttributes.getEmail());
+        assertEquals(expectedCreatedAt, observedAccountAttributes.createdAt);
+        assertEquals(expectedInstitute, observedAccountAttributes.getInstitute());
+        assertEquals(expectedIsInstructor, observedAccountAttributes.isInstructor());
+        assertEquals(expectedName, observedAccountAttributes.getName());
+        assertEquals(expectedGoogleId, observedAccountAttributes.studentProfile.googleId);
+    }
+
+    @Test
+    public void testBuilderWithUnsanitisedFieldValues() {
+        AccountAttributes observedAccountAttributes = AccountAttributes.builder()
+                .withGoogleId("googleId@gmail.com")
```

Comment:
Use `googleId@example.com` instead cause this might be an actual email id.
Update this everywhere

Reply:
The sanitisation code can only strip-off "@gmail.com".
Should I still change it to "@example.com"?

Reply:
Yes. Using "@example.com" is preferred over actual domains

Reply:
But then the test wont cover the functionality of stripping-off "@gmail.com" suffix.Is that ok?

Reply:
In that case let it be as it is. 👍